src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { PreparedPost } from '../../types/types';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo: React.FC<PreparedPost> = ({
+  title,
+  body,
+  user,
+  comments,
+}) => (
+  <li className="PostInfo" data-cy="post-info">
```

Comment:
Where is `key={post.id}`?

Reply:
The keys are used when component are called
![image](https://user-images.githubusercontent.com/57747481/166221687-831f51fe-d2ae-4f09-98a7-227178360746.png)